core/templates/filters/format-base-translatable-object-values.filter.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+  TranslatableSetOfNormalizedString, TranslatableSetOfUnicodeString
+} from 'interactions/rule-input-defs';
```

Comment:
The general pattern we have followed so far is to have imports below file overview.